rpa-manager/modules/ROOT/pages/processautomation-deploy.adoc

```diff
@@ -209,14 +236,18 @@ The bot can overtake the console or RDP session only if the specified user is lo
 Bots running an earlier version than 1.2.0 overtake an open console session without checking if the logged-in user matches the specified user.
 +
 [[runconfig-execution-enabletaskbar]] If you want to watch the execution of the process in the *Process Streaming* view of the *Process Monitoring* module or if the process uses elements of the Windows operating system like the start menu or the task bar, check *Enable Taskbar (Start menu, taskbar buttons)*.
-* *Process Trigger Execution*:
-+
-The productive run is carried out continuously within a fixed schedule. For each schedule, you can define time intervals within which the process is executed.
+* *Schedule*:
 +
+[%collapsible]
+.*Keep in mind that Bots might live in different time zones.*
```

Comment:
```suggestion
.*Note that bots might execute in different time zones.*
```
Removed the idiomatic expression and added "Note" to call the user's attention. Also wasn't sure about using "live" for the bot.

Reply:
Changed to 'Note' and 'run'